src/main/java/org/folio/service/inventory/InventoryItemRequestManager.java

```diff
@@ -0,0 +1,89 @@
+package org.folio.service.inventory;
+
+import static org.folio.service.inventory.InventoryUtils.INVENTORY_LOOKUP_ENDPOINTS;
+import static org.folio.service.inventory.InventoryUtils.REQUESTS;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.okapi.common.GenericCompositeFuture;
+import org.folio.rest.core.RestClient;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.core.models.RequestEntry;
+import org.folio.service.CirculationRequestsRetriever;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+
+public class InventoryItemRequestManager {
+
+  private static final Logger logger = LogManager.getLogger(InventoryItemRequestManager.class);
+
+  private static final String REQUEST_CANCEL_MESSAGE = "Cancelling Request with id='{}'";
+  private static final String REQUEST_MOVE_MESSAGE = "Moving Request with id='{}' to item with id='{}'";
+  private static final String REQUEST_ENDPOINT = INVENTORY_LOOKUP_ENDPOINTS.get(REQUESTS) + "/%s";
+  private static final String REQUEST_MOVE_ENDPOINT = REQUEST_ENDPOINT + "/move";
+  private static final String REQUEST_JSON_KEY = "destinationItemId";
```

Comment:
https://github.com/folio-org/mod-orders/pull/929#discussion_r1587758084

Reply:
Created a separate enum for such fields, good point